docs/source/tutorials/NISAR/Nisar_access_read_visualize_public.ipynb

```diff
@@ -0,0 +1,1116 @@
+{
```

Comment:
<p>Note:</p><p>When running on ade.ops.maap-project.org, I receive the following error:</p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_328/357315073.py in &lt;module&gt;
----&gt; 1 from h5glance import H5Glance

ModuleNotFoundError: No module named 'h5glance'
</pre><p>To do: requirement pip install</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>